server/service/src/item/item_variant/delete.rs

```diff
@@ -44,6 +44,14 @@ pub fn delete_item_variant(
                 })
                 .collect::<Result<Vec<_>, RepositoryError>>()?;
 
+            activity_log_entry(
+                ctx,
+                ActivityLogType::ItemVariantDeleted,
+                Some(input.id.clone()),
```

Comment:
Unsure which to save here... if we want to show it with the overall logs in item... then will need some refactoring to get the item id

Reply:
No just here me thinks.. If we move all the logs to a 'log page' then this won't be an issue 👀  though ideally would want to save BOTH the item id and the variant id... 

No issue, just one for refactoring enums which I totally agree. Though would love a bigger refactor to make it more informative..